acquisitions/src/main/resources/thunderjet/mod-orders/features/parallel-update-order-lines-same-order.feature

```diff
@@ -0,0 +1,98 @@
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+    * def orderId = callonce uuid3
+    * def poLineId1 = callonce uuid4
+    * def poLineId2 = callonce uuid5
+    * def poLineId3 = callonce uuid6
+    * def poLineId4 = callonce uuid7
+    * def poLineId5 = callonce uuid8
+
+    * def createOrder = read('../reusable/create-order.feature')
+    * def createOrderLine = read('../reusable/create-order-line.feature')
+    * def openOrder = read('../reusable/open-order.feature')
+
+    * configure headers = headersAdmin
+
+    * callonce createFund { 'id': '#(fundId)'}
+    * callonce createBudget { 'id': '#(budgetId)', 'allocated': 10000, 'fundId': '#(fundId)'}
+    * callonce createOrder { id: "#(orderId)" }
+    * callonce createOrderLine { id: "#(poLineId1)", orderId: "#(orderId)", fundId: "#(fundId)" }
+    * callonce createOrderLine { id: "#(poLineId2)", orderId: "#(orderId)", fundId: "#(fundId)" }
+    * callonce createOrderLine { id: "#(poLineId3)", orderId: "#(orderId)", fundId: "#(fundId)" }
+    * callonce createOrderLine { id: "#(poLineId4)", orderId: "#(orderId)", fundId: "#(fundId)" }
+    * callonce createOrderLine { id: "#(poLineId5)", orderId: "#(orderId)", fundId: "#(fundId)" }
+    * callonce openOrder { orderId: "#(orderId)" }
+
+    * configure headers = headersUser
+
+  Scenario: Update line 1
+    Given path 'orders/order-lines', poLineId1
+    When method GET
+    Then status 200
+    * def poLine = $
+    * set poLine.cost.listUnitPrice = 11
+    Given path 'orders/order-lines', poLineId1
+    And request poLine
+    When method PUT
+    Then status 204
+
+  Scenario: Update line 2
+    Given path 'orders/order-lines', poLineId2
+    When method GET
+    Then status 200
+    * def poLine = $
+    * set poLine.cost.listUnitPrice = 12
+    Given path 'orders/order-lines', poLineId2
+    And request poLine
+    When method PUT
+    Then status 204
+
+  Scenario: Update line 3
+    Given path 'orders/order-lines', poLineId3
+    When method GET
+    Then status 200
+    * def poLine = $
+    * set poLine.cost.listUnitPrice = 13
+    Given path 'orders/order-lines', poLineId3
+    And request poLine
+    When method PUT
+    Then status 204
+
+  Scenario: Update line 4
+    Given path 'orders/order-lines', poLineId4
+    When method GET
+    Then status 200
+    * def poLine = $
+    * set poLine.cost.listUnitPrice = 14
+    Given path 'orders/order-lines', poLineId4
+    And request poLine
+    When method PUT
+    Then status 204
+
+  Scenario: Update line 5
+    Given path 'orders/order-lines', poLineId5
+    When method GET
+    Then status 200
+    * def poLine = $
+    * set poLine.cost.listUnitPrice = 15
+    Given path 'orders/order-lines', poLineId5
+    And request poLine
+    When method PUT
+    Then status 204
```

Comment:
we can use Scenario Outline to avoid repition, there is two variable polinid and unitPrice

Reply:
it's only a suggestion, definetly we can remain as is

Reply:
if they doesn't work in scenario outline, you can resolve comment, it is okay to remain as it was

Reply:
Turns out it scenario outline works in parallel too. I updated it.

Reply:
just for knowladge sharing, I thought this part avoid parallel testing. do you think even if with this annotation scenario outline works parallel
![image](https://github.com/folio-org/folio-integration-tests/assets/113523904/f9161aba-0d7f-4812-bd73-7d88ee2ad6a0)